js/src/components/AdvancedSettings.js

```diff
@@ -104,8 +89,8 @@ const MetaRobotsNoIndex = ( { noIndex, onNoIndexChange, editorContext, isPrivate
 							editorContext.postTypeNameSingular,
 						) }
 					onChange={ onNoIndexChange }
-					name={ "yoast_wpseo_meta-robots-noindex-react" }
```

Comment:
Reminder for myself to test whether this has an averse effect on the saving of data.